CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
 Add protocol in device provision if not provided (#652)
-Fix: timestamp is now included just in non empty (more than id and type) multienty entitites
\ No newline at end of file
+Fix: timestamp is now included just in non empty (more than id and type) multienty entitites
+Add: uses provisioned timezone to generate TimeInstant values (#672)
```

Comment:
Are timezones mentioned in documentation? I think the a description on how this functionality works would be a good addition (if not yet existing).

Reply:
Fixed with 07ae0f2a7ae383a35a28e9dcc00d9a827993d26b